website/content/specification/syntax/constraints.md

```diff
@@ -14,27 +14,25 @@ TODO: P3: Address issue https://github.com/usnistgov/metaschema/issues/325
 
 ## Enumerated values
 
-Additionally, flags may be constrained to a set of known values listed in advance.
+One kind of Metaschema constraint is restricting assembly, field, or flag value(s) with `allowed-values` enumerations. Metaschema processors MUST process `allowed-values` enumerations as defined with the `allow-other` attribute.
 
-This restriction can be either:
+1. When the `allow-other` attribute is defined as `allow-other="no"`, the processor MUST strictly validate content instances with enumerations: only defined `enum` values are valid for the given target(s).
+2. When the `allow-other` attribute is defined as `allow-other="yes"`, the processor MUST loosely validate content instances with enumerations: both `enum` values and other values are valid.
+3. When the `allow-other` attribute is not explicitly defined, the Metaschema processor MUST strictly validate content instances. The implied default is `allow-other="no"`.
```

Comment:
This could be defined with the first item.

When the `allow-other` attribute is defined as `allow-other="no"` or not explicitly defined, ...

Reply:
Changed in https://github.com/usnistgov/metaschema/pull/413/commits/68a1d8de51251f4c820cae877bb6f00fa1fa1f60.